lib/jnpr/junos/utils/sw.py

```diff
@@ -984,11 +984,17 @@ def _progress(report):
                     kwargs.update({"no_validate": True})
 
             if issu is True:
+                if validate is False:
```

Comment:
Why are you explicitly setting the validate option as True always?

Are there any reference document , validation always be "True" in case of ISSU and NSSU

Reply:
when we set the option validate=False, we are not passing on this option to request system software add/request system software nonstop-upgrade/request system software add.

no-validate	
(Optional) When loading a software package or bundle with a different release, suppress the default behavior of the validate option.

validate	
(Optional) Validate the software package or bundle against the current configuration as a prerequisite to adding the software package or bundle. This is the default behavior when the software package or bundle being added is a different release.

no-validate	
(Optional) Starting in Junos OS Release 20.4R2, skip the validation step of the unified ISSU process.